fahrschein-http-jdk11/src/main/java/org/zalando/fahrschein/http/jdk11/JavaNetResponse.java

```diff
@@ -0,0 +1,101 @@
+package org.zalando.fahrschein.http.jdk11;
+
+import org.zalando.fahrschein.http.api.Headers;
+import org.zalando.fahrschein.http.api.HeadersImpl;
+import org.zalando.fahrschein.http.api.Response;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.http.HttpResponse;
+import java.util.zip.GZIPInputStream;
+
+final class JavaNetResponse implements Response {
+
+    private final HttpResponse<InputStream> r;
+
+    JavaNetResponse(HttpResponse<InputStream> r) {
+        this.r = r;
+    }
+
+    @Override
+    public int getStatusCode() throws IOException {
+        return this.r.statusCode();
+    }
+
+    @Override
+    public String getStatusText() throws IOException {
```

Comment:
This is interesting: java.net httpclient doesn't provide the http status line from the server at all. For backwards compat, we're re-injecting the strings for the common-known http status codes.

Reply:
Is it time to deprecate getStatusText() ?